googlecloud-plugins/src/main/java/co/cask/hydrator/plugin/sink/GCSAvroBatchSink.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin.sink;
+
+
+import co.cask.cdap.api.annotation.Description;
+import co.cask.cdap.api.annotation.Name;
+import co.cask.cdap.api.annotation.Plugin;
+import co.cask.cdap.api.data.batch.OutputFormatProvider;
+import co.cask.cdap.api.data.format.StructuredRecord;
+import co.cask.cdap.api.dataset.lib.KeyValue;
+import co.cask.cdap.etl.api.Emitter;
+import co.cask.cdap.etl.api.batch.BatchRuntimeContext;
+import co.cask.cdap.etl.api.batch.BatchSink;
+import co.cask.cdap.etl.api.batch.BatchSinkContext;
+import co.cask.hydrator.plugin.common.StructuredToAvroTransformer;
+import com.google.common.collect.Maps;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.mapred.AvroKey;
+import org.apache.avro.mapreduce.AvroKeyOutputFormat;
+import org.apache.hadoop.io.NullWritable;
+import org.apache.hadoop.mapred.JobContext;
+import org.apache.hadoop.mapreduce.lib.output.FileOutputFormat;
+
+import java.util.Map;
+
+/**
+ * {@link GCSAvroBatchSink} that stores data in avro format to Google Cloud Storage.
+ */
+@Plugin(type = BatchSink.PLUGIN_TYPE)
+@Name("GCSAvro")
+@Description("Batch sink to write to Google Cloud Storage in Avro format.")
+public class GCSAvroBatchSink extends GCSBatchSink<AvroKey<GenericRecord>, NullWritable> {
+
+  private StructuredToAvroTransformer recordTransformer;
+  private final GCSAvroSinkConfig config;
+
+  private static final String SCHEMA_DESC = "The Avro schema of the record being written to the sink as a JSON " +
```

Comment:
don't need a static final for this. Just use the string literal in the description field.